CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+## [UNRELEASED]
+### Added
+- Added a changelog
+
+### Changed
+- create orders without a contract_id
+- add get operation for orders
```

Comment:
well it depends. 
* If added is meant to only hold new files added to the project -> no.
* If added should contain all functionality added to the system regardless if new file or not -> yes. I opted for the first meaning.